jenkins/debian/Jenkinsfile_base

```diff
@@ -0,0 +1,17 @@
+library identifier: 'jenkins-libs@master', retriever: modernSCM(
+  [$class: 'GitSCMSource',
+   remote: 'https://github.com/ana/kernelci-jenkins-libs'])
+
+buildTest {
+
+    name = 'base'
+    archList = ["arm64" , "armel", "x86", "x86_64"]
+    debosFile = "jenkins/debian/debos/stretch.yaml"
```

Comment:
Wonder if we aren't leaking too much detail here and the Jenkinsfile shouldn't just say "stretch".

Or maybe the Jenkinsfile shouldn't have a say on the debian distro to use as a base. I think I would start by hiding it completely in the shared library, and only exposing it to the Jenkinsfiles if absolutely needed in the future.